### ISSUE TITLE
Resnet-50 trained models

### ISSUE THREAD

**Issue by kwea123** [nianticlabs/monodepth2#19]
Hi, are the models listed [here](https://github.com/nianticlabs/monodepth2#%EF%B8%8F-prediction-for-a-single-image) all with resnet-18 backbone? Do you also provide resnet-50 trained ones (the one in the paper page 12 table 6c)?

Thank you very much.

**Comment by kwea123** [comment]
Also, can you also provide the weights for predictive masks for visualization purpose?

**Comment by mdfirman** [comment]
Thanks for the suggestion. 

Here are resnet50 weights for Imagenet pretrained:
https://storage.googleapis.com/niantic-lon-static/research/monodepth2/mono_resnet50_640x192.zip

And trained from scratch:
https://storage.googleapis.com/niantic-lon-static/research/monodepth2/mono_resnet50_no_pt_640x192.zip

I have also added links to these models to the README

**Comment by mdfirman** [comment]
Here are the weights for the predictive mask baseline:
https://storage.googleapis.com/niantic-lon-static/research/monodepth2/predictive_mask_model.zip
